src/Project/MvpSite/rendering/Mvp.Project.MvpSite.Rendering.csproj

```diff
@@ -23,6 +23,7 @@
 
 	<ItemGroup>
 	  <Content Remove="compilerconfig.json" />
+	  <Content Remove="Views\Shared\NotFound.cshtml" />
```

Comment:
This shouldn't be needed as we're hosting in PaaS not IaaS, the app is deployed fresh each time so there will be no legacy files to remove